content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/100-cyclic-referential-actions.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Cyclic referential actions'
+metaTitle: 'Cyclic referential actions'
+metaDescription: 'Circular references can occur between models when each model references another creating a closed loop. Learn how Prisma identifies and suggests solutions to this potential problem.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Circular references can occur between models when each model references another creating a closed loop. Most SQL databases handle these references out of the box. 
+
+With SQLServer however, if the [referential action](./) on the related field is set to something other than `NoAction`, the data model validator will throw an error.
```

Comment:
I would add a mention about the fact that it only applies for cyclic relations.
I also wonder about the term "related field" as I wonder which field we specifically mean here. I guess it could help to use "relation field" or something more precise?